platforms/native/platform-native/src/testFixtures/groovy/org/gradle/nativeplatform/fixtures/AvailableToolChains.java

```diff
@@ -78,18 +78,17 @@ public int compare(ToolChainCandidate toolchain1, ToolChainCandidate toolchain2)
     private static List<ToolChainCandidate> toolChains;
 
     /**
-     * Locates the tool chain that would be used as the default for the current machine, if any.
-     *
-     * @return null if there is no such tool chain.
+     * Locates the C++ tool chain that would be used as the default for the current machine. Asserts that there is a tool chain installed.
      */
-    @Nullable
-    public static InstalledToolChain getDefaultToolChain() {
-        for (ToolChainCandidate toolChain : getToolChains()) {
-            if (toolChain.isAvailable()) {
-                return (InstalledToolChain) toolChain;
+    public static InstalledCppToolChain getDefaultToolChain() {
```

Comment:
Every test that uses this method expects it to return a C++ compiler (rather than, say, `null` or the Swift compiler), so change the contract to make this explicit.